tests/bson/bson-decode-001.phpt

```diff
@@ -62,7 +62,7 @@ Test#2 { "0" : null }
 OK
 Test#3 { "0" : 123 }
 OK
-Test#4 { "0" : 4.23 }
+Test#4 { "0" : 4.23%d }
```

Comment:
AFAIK, this shouldn't be necessary if you bump libbson. Older versions printed more precision than was necessary.

Also, `%d` doesn't have a "zero or more" variant (like `%s` and `%S`), so this could fail for newer libbson versions that only print `4.23`, no?